include/ifc/index-utils.hxx

```diff
@@ -165,6 +165,10 @@ namespace index_like {
     template<typename S>
     constexpr auto index_precision = 32 - tag_precision<S>;
 
+    // Squelch -Wconversion
+    template<unsigned Precision>
+    constexpr auto bit_field_mask = ~((std::numeric_limits<std::size_t>::max)() << Precision);
```

Comment:
Why `std::size_t` instead of `std::uint32_t`?

Reply:
No particular reason. Changed.